packages/app-form-builder/src/admin/components/FormEditor/Tabs/EditTab/Field.tsx

```diff
@@ -29,26 +30,70 @@ const Actions = styled("div")({
     alignItems: "right",
     "> *": {
         flex: "1 100%"
+    },
+    ".switch-wrapper": {
+        display: "flex",
+        alignItems: "center",
+        color: "var(--mdc-theme-text-secondary-on-background)",
```

Comment:
Just extra clarification here.

The usage of SCSS vars here is OK.

This is because these are Admin app styles, and these are still handled via SCSS.

Only Admin app styles and theme vars are handled via SCSS. The rest (Page Builder pages, Form Builder forms) is handled via the new `theme.ts` file.

If you'll have any extra questions, feel free to ping.